layouts/community/ergodox/nstickney/keymap.c

```diff
@@ -0,0 +1,221 @@
+#include QMK_KEYBOARD_H
+
+/*
+ * This keymap simulates the key limitations of an Iris on an ErgoDox. See the
+ * matching iris layout (/keyboards/iris/keymaps/nstickney) for further
+ * information.
+ */
+
+// Layers
+#define BASE 0 // Base layer
+#define SYMB 1 // Symbols
+#define SYSH 2 // Symbols, shifted
+#define NUMP 4 // Numpad
+#define FCTN 8 // Function
+
+bool _layer_active(uint8_t layer) {
```

Comment:
Actually, you could use `IS_LAYER_ON` 

https://github.com/qmk/qmk_firmware/blob/c5221fa1cb1e903600205ba831c841f9d3aad33f/quantum/quantum.h#L220

Reply:
I figured there was something like this available but it looks like I was searching the wrong things to find it.